_data/simple-icons.json

```diff
@@ -5692,6 +5692,11 @@
             "hex": "1273EB",
             "source": "https://commons.wikimedia.org/wiki/File:Freepik.svg"
         },
+        {
+            "title": "FreeTube",
+            "hex": "000000",
```

Comment:
This doesn't represent the brand color. From what I can tell it should be some sort of red or blue, you can find the correct shades/hexes in https://github.com/FreeTubeApp/FreeTube/blob/development/_icons/iconColor.svg.

Reply:
There are other uncolored logos in this project, and I don't want to chose just one color for a two color logo.

Reply:
That's not how this works, all icons have an appropriate color for the brand/product identity. See https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#color-guidelines.

Reply:
I agree with @uncenter - if the icon contains multiple colours we pick the most prominent (in this case, the red) or the midpoint of all point colours (in the case of a gradient).